recipe/meta.yaml

```diff
@@ -26,6 +26,7 @@ requirements:
   build:
     - r-base
     - r-rcpp
```

Comment:
The [DESCRIPTION](https://github.com/tidyverse/tibble/blob/v1.3.3/DESCRIPTION) for version 1.3.3 states r-rccp >=0.12.3